README.md

```diff
@@ -515,54 +639,44 @@ The `ucan` scheme MUST accept the following ability: `ucan/*`. This ability rede
 If an attenuated resource or capability is desired, it MUST be explicitly listed without the `ucan` URI scheme.
 
 ``` js
-[
-  { 
-    "with": "ucan:bafkreihogico5an3e2xy3fykalfwxxry7itbhfcgq6f47sif6d7w6uk2ze",
-    "can": "ucan/*"
-  }, 
-  { 
-    "with": "ucan:*", 
-    "can": "ucan/*" 
-  }
-]
+{ 
+  "ucan:bafkreihogico5an3e2xy3fykalfwxxry7itbhfcgq6f47sif6d7w6uk2ze": {"ucan/*": [{}]}, 
+  "ucan:*": {"ucan/*": [{}]}
```

Comment:
Do we need `ucan/` prefix on the ability here ? We used to have special `*` ability to denote "all" abilities which was far more intuitive than `ucan/*`. If it really need to be name-spaced I would  suggest `*/*` as it feels far more intuitive that `*` and `*/*` would include `crud/read` than `ucan/*`.

Reply:
We also use `{ with: "ucan:*", can: "*" }` when linking devices which I think would be considered invalid ? If so it is probably worth calling out what it would imply.